app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -37,13 +37,13 @@
   <div class="media-component-body">
     <aside class="left-drawer collapse" data-media-target="leftDrawer">
       <nav class="vert-tabs">
-        <button data-action="click->media#displayMetadata" data-media-target="leftButton" class="active">
+        <button data-action="click->media#displayMetadata" data-media-target="leftButton" class="active" aria-label="About this item">
           <svg class="MuiSvgIcon-root" focusable="false" viewBox="0 0 24 24" aria-hidden="true"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"></path></svg>
         </button>
-        <button data-action="click->media#displayRights" data-media-target="leftButton">
+        <button data-action="click->media#displayRights" data-media-target="leftButton" aria-label="Use and reproduction">
```

Comment:
I think this tab is more than Use and reproduction, but this is better than what w. have now.

Reply:
Open to any suggestions on a more descriptive label :-)